Deployment is returning 502: BAD_GATEWAY error

I am trying to sort out what is causing the error. Here is the error from the Build Logs:

```
12:52:57.034 | ┌──────────────────────────────┐
-- | --
12:52:57.035 | │ built server with 2 warnings │
12:52:57.035 | └──────────────────────────────┘
12:52:57.035 | > 'images/successkid.jpg' is imported by src/routes/index.svelte, but could not be resolved – treating it as an external dependency
12:52:57.035 | > Entry module "src/server.ts" is implicitly using "default" export mode, which means for CommonJS output that its default export is assigned to "module.exports". For many tools, such CommonJS output will not be interchangeable with the original ES module. If this is intended, explicitly set "output.exports" to either "auto" or "default", otherwise you might want to consider changing the signature of "src/server.ts" to use named exports only.
```

I am thinking the 2nd error could be the main problem with the deploy. Here is what my server.ts file shows:

```
import sirv from 'sirv';
import polka from 'polka';
import compression from 'compression';
import * as sapper from '@sapper/server';

const { PORT, NODE_ENV } = process.env;
const dev = NODE_ENV === 'development';

export default polka() // You can also use Express
	.use(
		compression({ threshold: 0 }),
		sirv('static', { dev }),
		sapper.middleware()
	)
	.listen(PORT, err => {
		if (err) console.log('error', err);
	});

```

However, when I run `vercel` in the CLI I get this error:
```
Due to `builds` existing in your configuration file, the Build and Development Settings defined in your Project Settings will not apply.
```

I assume that the mentioned "configuration file" is vercel.json. Mine looks like this:

```
{
  "version": 2,
  "builds": [
    {
      "src": "package.json",
      "use": "vercel-sapper"
    }
  ]
}
```

Does anyone have guidance on how to fix the 502: BAD_GATEWAY error I am experiencing? Thank you in advance for any assistance.

I did not end up needing to reach out to Vercel. After taking a better look at my Function Logs I noticed that the image folder where successkid.jpg is stored was throwing repeated errors (at one point 11 errors). I did find it odd that the image folder was stored in node_modules...Should it really be there? For some reason it could never resolve the image module. Here is the error:

```
[GET] /
19:47:41:87
2020-10-19T02:47:42.081Z	undefined	ERROR	Server is not listening Error: Cannot find module 'images/successkid.jpg'Require stack:- /var/task/__sapper__/build/server/server.js- /var/task/launcher.js- /var/runtime/UserFunction.js- /var/runtime/index.js    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)    at Function.Module._load (internal/modules/cjs/loader.js:841:27)    at Module.require (internal/modules/cjs/loader.js:1025:19)    at require (internal/modules/cjs/helpers.js:72:18)    at Object.<anonymous> (/var/task/__sapper__/build/server/server.js:8:18)    at Module._compile (internal/modules/cjs/loader.js:1137:30)    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)    at Module.load (internal/modules/cjs/loader.js:985:32)    at Function.Module._load (internal/modules/cjs/loader.js:878:14)    at Module.require (internal/modules/cjs/loader.js:1025:19) {  code: 'MODULE_NOT_FOUND',  requireStack: [    '/var/task/__sapper__/build/server/server.js',    '/var/task/launcher.js',    '/var/runtime/UserFunction.js',    '/var/runtime/index.js'  ]}RequestId: f9708333-685f-4392-9de1-ae546c3fa7d6 Error: Runtime exited with error: exit status 1Runtime.ExitError
```
I ended up deleting the image and all related code from the index.svelte file and presto no more 502 Bad Gateway error
**index.svelte file**
```
<script>

</script>

<style>
	h1, figure, p {
		text-align: center;
		margin: 0 auto;
	}

	h1 {
		font-size: 2.8em;
		text-transform: uppercase;
		font-weight: 700;
		margin: 0 0 0.5em 0;
	}

	figure {
		margin: 0 0 1em 0;
	}

	img {
		width: 100%;
		max-width: 400px;
		margin: 0 0 1em 0;
	}

	p {
		margin: 1em auto;
	}

	@media (min-width: 480px) {
		h1 {
			font-size: 4em;
		}
	}
</style>

<svelte:head>
	<title>Sapper project template</title>
</svelte:head>

<h1>Great success!</h1>

<p><strong>Try editing this file (src/routes/index.svelte) to test live reloading.</strong></p>

```

--- a/bin.js
+++ b/bin.js
@@ -96,9 +96,10 @@ async function patch(dir) {
     }
     fs.writeFileSync(
       join(dir, '.vercelignore'),
-      `__sapper__
-cypress
-node_modules`
+      `/__sapper__
+/cypress
+!node_modules
+/node_modules/*`
     )
     console.log(green('Created'), '.vercelignore')
   } catch (e) {

